CHANGELOG.md

```diff
@@ -1,8 +1,63 @@
+- improve checks & error messages to help integrators:
+  - show warnings in console when solution contains unrecognized keys
+  - show warnings in console when workspace contains unrecognized keys
+  - improved error messages context & format for PowerBI service account errors
+- more context shown to end-users:
+  - workspace information is now visible from scenario view
+  - scenario run types are now visible from scenario manager
+  - the technical details of the solution are shown in the "Help" menu
+  - simulation logs can now be downloaded even when scenario runs have succeeded
+- and more new features:
+  - when option `shouldRenameFileOnUpload` is set for files or tables parameters, uploaded files are renamed before being saved
+  - add optional tooltips to describe values of "enum" scenario parameters
+  - add translation support for enum values
+
+### Bug Fixes
+
+- force date values received from \(and sent to\) back\-end to be set to midnight UTC ([987f1cf](https://github.com/Cosmo-Tech/azure-sample-webapp/commit/987f1cfb))
+- rename \.xlsx files imported in Table to \.csv before upload ([5e75dfc](https://github.com/Cosmo-Tech/azure-sample-webapp/commit/5e75dfc0))
+- \[PROD\-9531\] prevent aggrid from automatically renaming table columns ([9fcb439](https://github.com/Cosmo-Tech/azure-sample-webapp/commit/9fcb4390))
+- \[PROD\-11247\] make email & roles non\-case\-sensitive ([25ececb](https://github.com/Cosmo-Tech/azure-sample-webapp/commit/25ececbd))
+- \[PROD\-12144\] make running state more visible ([d07c6bc](https://github.com/Cosmo-Tech/azure-sample-webapp/commit/d07c6bcd))
+- sligthly increase size of run template name in Scenario view ([1c18af0](https://github.com/Cosmo-Tech/azure-sample-webapp/commit/1c18af02))
+- \[PROD\-7940\] sort scenarios in dropdown list, in scenario creation pop-up ([9d23168](https://github.com/Cosmo-Tech/azure-sample-webapp/commit/9d231682))
+- \[PROD\-11951\] fix errors message mentioning "scenario run" when the "Save" action failed ([8efa3da](https://github.com/Cosmo-Tech/azure-sample-webapp/commit/8efa3da6))
+- disable folding of organization Accordion in workspaces selector ([834f557](https://github.com/Cosmo-Tech/azure-sample-webapp/commit/834f5576))
+- \[PROD\-12104\] running scenarios are now stopped before being deleted ([2bae6fc](https://github.com/Cosmo-Tech/azure-sample-webapp/commit/2bae6fc0))
+- show error in browser console when conversions of parameter values have failed ([08ff30f](https://github.com/Cosmo-Tech/azure-sample-webapp/commit/08ff30f0))
+- fix possible errors in Powerbi reducer & saga ([fcd1929](https://github.com/Cosmo-Tech/azure-sample-webapp/commit/fcd1929d))
+- fix broken behavior of REACT_APP_NO_POWERBI_POLLING option ([be87a09](https://github.com/Cosmo-Tech/azure-sample-webapp/commit/be87a093))
+- and more minor bug fixes...
 
 ### Known issues
 
 - clearing the errors panel of a Table parameter does not reset the "dirty" state of the scenario parameters form
```

Comment:
Maybe we should reformulate the behavior issue to be more factual : "Clearing the errors panel of a table parameter causes systematically dirty state as true".

Reply:
As discussed, keeping this version for now :slightly_smiling_face: